packages/manager/apps/web/client/app/domain/anycast/terminate/terminate.state.js

```diff
@@ -0,0 +1,29 @@
+const state = {
+  url: '/anycast-terminate',
+  views: {
+    domainView: {
+      component: 'domainAnycastTerminate',
+    },
+  },
+  resolve: {
+    previousState: /* @ngInject */ ($transition$) => $transition$.$from(),
+    dnsAnycastDetails: /* @ngInject */ (Domain, domainName) =>
+      Domain.getDnsAnycastDetails(domainName),
+    setError: /* @ngInject */ (Alerter) => (message) =>
+      Alerter.error(message, 'domain.terminateDnsAnycast'),
+    goBack: /* @ngInject */ ($state, previousState) => () => {
+      if (previousState.name) {
+        $state.go(previousState.name);
+      } else {
+        $state.go('app.domain.product.information');
+      }
```

Comment:
I think at least you have to write `return` statement.

```suggestion
   return $state.go(previousState.name ? previousState.name : 'app.domain.product.information');
```